staking/asn1.go

```diff
@@ -14,11 +15,28 @@ import (
 	_ "crypto/sha256"
 )
 
-// Ref: https://github.com/golang/go/blob/go1.19.12/src/crypto/x509/x509.go#L326-L350
-var signatureAlgorithmVerificationDetails = map[x509.SignatureAlgorithm]x509.PublicKeyAlgorithm{
-	x509.SHA256WithRSA:   x509.RSA,
-	x509.ECDSAWithSHA256: x509.ECDSA,
-}
+var (
+	// Ref: https://github.com/golang/go/blob/go1.19.12/src/crypto/x509/x509.go#L433-L452
```

Comment:
nit: should we use the permalink here? `https://github.com/golang/go/blob/0ae54ddd37302bdd2a8c775135bf5f076a18eeb3/src/crypto/x509/parser.go#L789-L968`

Reply:
The link as is should be a perma link (as long as the repo doesn't change what the tag `v1.19.12` points to... which wouldn't make much sense to me)